doc/leak_tracking/DETECT.md

```diff
@@ -5,12 +5,42 @@ The text below is under construction.
 
 # Detect Memory Leaks
 
-This page describes how to auto-detect certain types of memory leaks in Dart and Flutter applications and tests.
-See other information on memory leaks [here](../README.md).
+This page describes how to auto-detect certain types of memory leaks.
+Read more about leak tracking in [overview](OVERVIEW.md).
 
 ## Quick start to track leaks for Flutter
 
-### Flutter application
+### Test cover with `testWidgets`
+
+To make `testWidgets` detecting leaks in your tests, enable leak tracking in [`test/flutter_test_config.dart`](see hhttps://api.flutter.dev/flutter/flutter_test/flutter_test-library.html):
+
+```dart
+import 'dart:async';
+
+import 'package:leak_tracker_flutter_testing/leak_tracker_flutter_testing.dart';
+
+...
+
+FutureOr<void> testExecutable(FutureOr<void> Function() testMain) {
+  LeakTesting.enable();
+  LeakTesting.settings = LeakTesting.settings
+    .withIgnored(createdByTestHelpers: true);
+
+  ...
+
+  return testMain();
+}
+```
+
+See [documentation for `testWidgets`](https://github.com/flutter/flutter/blob/4570d35d49477a53278e648ce59a26a06201ec97/packages/flutter_test/lib/src/widget_tester.dart#L122) on how to adjust leak tracking settings.
+
+### Instrument more disposables
+
+Use [this example](https://github.com/flutter/flutter/pull/138804/files) to add your disposables to leak tracking.
```

Comment:
To instrument a disposable class for leak tracking, you need to dispatch object creation and disposal events to leak tracker. Use [this example](https://github.com/flutter/flutter/pull/138804/files) as a guide.